### ISSUE TITLE
Ignore some elements

### ISSUE THREAD

**Issue by fernandojsg** [patrickkunka/mixitup#223]
Hi everyone,
I've been trying mixitup for a while and I couldn't find a way to easy ignore some elements. 
Currently I'm using it for showing my projects portfolio: http://fernandojsg.com/projects/ but I wanted to hide by default old projects so I wanted to add something like  "[x] Include old projects". My idea was to do something like find the "old" class and hide them by default, but the filter condition if stronger than my hiding :)
In summary I want to add/remove the old projects to the panel so the filter will act on them all or all-old.
Any suggestion how to make it or if it's currently supported this kind of behaviour?

**Comment by patrickkunka** [comment]
Can you use the css `:not()` selector?

`<div class="filter" data-filter=".category-1:not(.old)"></div>`

**Comment by fernandojsg** [comment]
@patrickkunka the point is that for example I could have this:
[category1] [category2] [ ]show old

If I click category1 it will show just category1 from non-old project, but If I select show-old, it must show category1 old and new projects. So I guess maybe dynamically changing :not(.old) ? adding and removing it from data-filter could work.
Something like if "show-old" clicked: data-filter=".category-1", if "show-old" non clicked: data-filter=".category-1:not(.old"
I'm going to try it

**Comment by fernandojsg** [comment]
@patrickkunka yep that worked! I just toggle the :not(.old) for each filter depending on the "[x] include old" toggle :)
Just a final question, is there any way to actually refresh the filtering? If I add dynamically the :not(.old) selector to a filter and that filter is currently selected, to retrigger the ordering with the modified filter?
